common/util.go

```diff
@@ -25,6 +25,14 @@ type ConnectorMetadata struct {
 	SourceName    string
 }
 
+// GCPPubsubConnInfo contains the fields needed to connect to the GCP's pub sub topic queue.
+type GCPPubsubConnInfo struct {
```

Comment:
Common package is common code used by various message queues. This code specific to GCP please move this to gcp connector code.